src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -202,46 +159,157 @@ public boolean wristAtSetpoint() {
         return wristPID.atSetpoint();
     }
 
-    @Override
-    public void initSendable(SendableBuilder builder) {
-        super.initSendable(builder);
-        builder.addDoubleProperty("WristGoalPos", () -> goalPosRad[1], goal -> goalPosRad[1] = goal);
-        builder.addDoubleProperty("ArmPos", () -> getArmPos(), null);
-        builder.addDoubleProperty("WisPos", () -> getWristPos(), null);
-        builder.addDoubleProperty("RobotWisPos", () -> getWristPosRelativeToGround(), null);
-        builder.addDoubleProperty("kP: Arm", () -> kP[ARM], x -> kP[ARM] = x);
-        builder.addDoubleProperty("kP: Wis", () -> kP[WRIST], x -> kP[WRIST] = x);
-        builder.addDoubleProperty("kI: Arm", () -> kI[ARM], x -> kI[ARM] = x);
-        builder.addDoubleProperty("kI: Wis", () -> kI[WRIST], x -> kI[WRIST] = x);
-        builder.addDoubleProperty("kD: Arm", () -> kD[ARM], x -> kD[ARM] = x);
-        builder.addDoubleProperty("kD: Wis", () -> kD[WRIST], x -> kD[WRIST] = x);
-    }
-
     public boolean isCubeMode() {
-        return isCube == 0;
+        return object == CUBE;
     }
+
     public boolean isFrontMode() {
         return isFront;
     }
+
     public void toggleCube() {
-        isCube++;
-        isCube %= 2;
+        object++;
+        object %= 2;
     }
+
     public void toggleFront() {
         isFront = !isFront;
     }
+
     public double getArmGoal(GoalPos[] pos) {
         if (isFront) {
-            return pos[isCube].armPos;
+            return pos[object].armPos;
         } else {
-            return Rotation2d.fromRadians(-Math.PI).minus(Rotation2d.fromRadians(pos[isCube].armPos)).getRadians();
+            return Rotation2d.fromRadians(-Math.PI).minus(Rotation2d.fromRadians(pos[object].armPos)).getRadians();
         }
     }
+
     public double getWristGoal(GoalPos[] pos) {
         if (isFront) {
-            return pos[isCube].wristPos;
+            return pos[object].wristPos;
         } else {
-            return Rotation2d.fromRadians(pos[isCube].wristPos).unaryMinus().getRadians();
+            return Rotation2d.fromRadians(pos[object].wristPos).unaryMinus().getRadians();
         }
     }
```

Comment:
Just call `setArmTarget` and `setWristTarget` with the required `GoalPos` object. If anything, you should have no more than a `getGoalPos()` method.

Reply:
These will dissapear with the OI rewrite
need someone to finish it